### PR TITLE
SDKS-300: Incorrect treatment result

### PR DESCRIPTION
**Pull request by javrudsky**
Workaround applied to fix a cast from Any to Int64 that was causing an incorrect evaluation.

### CODE REVIEW

File: Split/Common/Utils/CastUtils.swift [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+
+class CastUtils {
+    static func anyToInt64(value: Any?) -> Int64? {
+        if let val = value as? NSNumber {
+            if val.intValue <= Int64.max {
+                return Int64(truncating: val)
+            }
+        }
+        return nil
```
Comment (SenhorCastor) [inline Split/Common/Utils/CastUtils.swift]: are we ok returnin nill?

File: Split/Matchers/GreaterThanOrEqualToMatcher.swift [hunk 2]
```diff
@@ -37,7 +37,7 @@ public class GreaterThanOrEqualToMatcher: BaseMatcher, MatcherProtocol {
                 return  attributeDate >= backendDate
```
Comment (SenhorCastor) [inline Split/Matchers/GreaterThanOrEqualToMatcher.swift]: Can we have some sort of unit test for all these? (not this file but all the classes affected)

File: Split/Common/Utils/CastUtils.swift (hunk 1 above)
Comment (javrudsky) [reply Split/Common/Utils/CastUtils.swift]: I think is the best option for the case the cast fails. What do you think about that? Do you have any suggestion?

File: Split/Matchers/GreaterThanOrEqualToMatcher.swift (hunk 2 above)
Comment (javrudsky) [reply Split/Matchers/GreaterThanOrEqualToMatcher.swift]: Sure, will do!

**Pull request merged**